src/testdriver.c

```diff
@@ -62,7 +62,7 @@ size_t print_string(char* buf, size_t len, stoken_t* t)
 
 size_t print_var(char* buf, size_t len, stoken_t* t)
 {
-    int slen = 0;
+    int slen;
```

Comment:
I would suggest to reduce the scope of slen as oppose to removing the initial attribution. But that mitigates the issue as well.